content/post/2.0.0.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Release 2.0.0 is available
+date: 2021-03-24
+type: release
+linked: true
+---
+<!---
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License. See accompanying LICENSE file.
+-->
+
+[Download](https://ratis.apache.org/downloads.html)
+
+It contains around 260 improvements and bug fixes based on various Apache Hadoop Ozone use cases.
+See the [changes between 1.0.0 and 2.0.0](https://github.com/apache/ratis/compare/ratis-1.0.0...ratis-2.0.0-rc4) releases.
```

Comment:
Please create a ratis-2.0.0 tag.  We should not use the rc4 tag for the official release.